umap/views.py

```diff
@@ -634,6 +668,20 @@ def post(self, *args, **kwargs):
         return response
 
 
+class MapStar(View):
```

Comment:
```suggestion
class ToggleMapStarStatus(View):
```

For a more explicit name (URLs need to be updated accordingly).

Reply:
Done!